builtin/logical/pki/cert_util.go

```diff
@@ -571,6 +572,20 @@ func generateCreationBundle(b *backend,
 		}
 	}
 
+	// Set OU (organizationalUnit) values if specified in the role
+	ou := []string{}
+	{
+		if role.OU != "" {
```

Comment:
Can we make the `OU` field of `roleEntry` a `[]string` instead of a string? Also, we can use the inbuilt helper function `strutil.ParseDedupAndSortStrings()` to sanitize the input before storing in the role. That way, the logic here translates to `ou := role.OU`.

The reason for suggesting this change is to avoid having to parse the role's fields in all the places they are retrieved.